Examples/BugReport/Tests/BugReportTests/BugReportTests.swift

```diff
@@ -0,0 +1,14 @@
+import MongoSwift
+
+import XCTest
+import class Foundation.Bundle
```

Comment:
do we need this import?

Reply:
Seems like a no. This was an artifact from `swift package init`.